packages/atomic/src/components/search/generated-answer/atomic-generated-answer/atomic-generated-answer.tsx

```diff
@@ -96,6 +99,7 @@ export class AtomicGeneratedAnswer implements InitializableComponent {
 
   private storage: SafeStorage = new SafeStorage();
   private data?: GeneratedAnswerData;
+  private showCopyToClipboard: boolean = false;
```

Comment:
We could do just a getter function
```suggestion
   private get hasClipboard() {
     return !!navigator?.clipboard?.writeText;
   }
```